qcelemental/models/molecule.py

```diff
@@ -773,7 +780,7 @@ def from_file(cls, filename: str, dtype: Optional[str] = None, *, orient: bool =
                 dtype = "string"
 
         # Raw string type, read and pass through
-        if dtype in ["string", "xyz", "psi4"]:
+        if dtype in ["string", "xyz", "xyz+", "psi4"]:
```

Comment:
now you point it out, should probably drop `psi4+` from https://github.com/MolSSI/QCElemental/pull/152/files#diff-c1df0b88317a0c0331bea59a1793059cR715, too. it will only collect a spec that `model.Molecule` can't handl.e